DXMainClient/DXGUI/Multiplayer/GameLobby/CnCNetGameLobby.cs

```diff
@@ -1593,6 +1605,16 @@ private void MapSharer_HandleMapDownloadComplete(SHA1EventArgs e)
                     ChangeMap(GameModeMap);
                 }
             }
+            else if (chatCommandDownloadedMaps.Contains(e.SHA1))
+            {
+                // Somehow the user has managed to download an already existing sha1 hash.
+                // This special case prevents user confusion by the file successfully downloading but showing an error.
+                AddNotice(returnMessage, Color.Yellow);
+                AddNotice(
+                    "Map was downloaded via chat, but a duplicate of the map with a separate filename is already loaded. This may cause strange behavior.",
```

Comment:
Needs a L10N call here, also not sure if we need to split the line like this.

Reply:
My bad, the line was long and I'm used to python where we aggressively split lines. I'll remove it and add the internationalization calls.

Reply:
No worries, check the other changes, IIRC there was a line like that too. It's OK to split lines IIRC, just need to add a newline after them and leave the closing bracket at last argument line IIRC (consult the [styleguide](https://github.com/CnCNet/xna-cncnet-client/blob/develop/Contributing.md) if unsure).